src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { PersonItem } from '../PersonItem/PersonItem';
+
+type Props = {
+  people: Person[];
+  isError: boolean;
+  isLoading: boolean;
+};
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  isError,
+  isLoading,
+}) => {
+  return (
+    <div className="block">
+      <div className="box table-container">
+        {isLoading && <Loader />}
+        {!isLoading && !isError && (
```

Comment:
we can try to rewrite it all using ternary operator

```
{isLoading ? (<Loading />) : (
   <>
    // ...
   </>
)}
```

Reply:
i get a lot of "Do not nest ternary expressions" errors , while trying to make it that way(
is it really bad to make it like it was done before?